src/Command/InitCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Command;
+
+use App\Entity\User;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class InitCommand extends Command
+{
+    private UserPasswordEncoderInterface $passwordEncoder;
+    private ManagerRegistry $registry;
+
+    protected static $defaultName = 'starfleet:init';
```

Comment:
This command will become a more complete command : init admin + init xxx + init yyy?

"Init" seems a little big too generic for just an account creation.

Reply:
Yes, this command will become more complex. I plan to add initialization for some mandatory settings (slack token, tags, etc)